docs/src/routes/docs/components/dot.svx

```diff
@@ -0,0 +1,86 @@
+---
+name: Dot
+---
+
+<script>
+  import { Dot, Button } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+</script>
+
+# Dot
+
+A dot.
```

Comment:
You, sir, get an F for expressiveness)
Maybe we could play around with this, something like "a colorful dot to signal about all your important events"